### PR TITLE
create applier process to install exporters

### PR DESCRIPTION
**Pull request by ramius345**
I added some applier code to automatically install the two exporters we have.  I was having trouble with the template and applier properly processing vartiables, so I switched it to a jinja template.  I needed to bring in the master version of applier (upping it from v2.1.1).

### CODE REVIEW

File: requirements.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # From 'openshift-applier'
 - src: https://github.com/redhat-cop/openshift-applier
-  version: v2.1.1
+  version: master
```
Comment (etsauer) [inline requirements.yml]: @ramius345 can you try applier v2.1.2?

File: README.md [hunk 2]
```diff
@@ -77,3 +80,23 @@ ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/o
 # Deploy MDT Tool
 ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e provision=false
 ```
+
```
Comment (etsauer) [inline README.md]: Let's put this above the cleanup section.

File: README.md [hunk 3]
```diff
@@ -77,3 +80,23 @@ ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/o
...
+### Deploying Exporters
+
+To deploy the exporters, first setup the following environment variables and create the following secret:
+
+```
+export GITHUB_REPOS=redhat-cop/mdt-quickstart
+export GITHUB_USER=<your github user name>
+export GITHUB_TOKEN=<your github token>
+oc process -f templates/github-secret.yaml -p GITHUB_USER=${GITHUB_USER} -p GITHUB_TOKEN=${GITHUB_TOKEN} -p GITHUB_REPOS=${GITHUB_REPOS}  | oc apply -f-
```
Comment (etsauer) [inline README.md]: need to add `-n custom-dashboards`

File: exporters-yaml/sa.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: robot
```
Comment (AlyIbrahim) [inline exporters-yaml/sa.yaml]: @ramius345 Should we give the Service Account a meaningful name other than robot?

File: README.md (hunk 3 above)
Comment (AlyIbrahim) [inline README.md]: Environment Variable `PROJECTS` is optionally needed by the lead time exporter

File: README.md [hunk 5]
```diff
@@ -77,3 +80,23 @@ ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/o
...
 ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e provision=false
 ```
+
+
+### Deploying Exporters
+
+To deploy the exporters, first setup the following environment variables and create the following secret:
+
+```
+export GITHUB_REPOS=redhat-cop/mdt-quickstart
```
Comment (AlyIbrahim) [inline README.md]: `GITHUB_REPOS` can be single value or a comma separated list

File: README.md (hunk 3 above)
Comment (AlyIbrahim) [reply README.md]: It is a comma separated list too

**Comment by etsauer** [comment]
@ramius345 we need a rebase here.

File: requirements.yml (hunk 1 above)
Comment (ramius345) [reply requirements.yml]: This needs to stay master until jinja_vars gets put into a release.

File: exporters-yaml/sa.yaml (hunk 4 above)
Comment (ramius345) [reply exporters-yaml/sa.yaml]: Changed to exporter-robot

File: README.md (hunk 3 above)
Comment (ramius345) [reply README.md]: I added a separate config map to handle this.

File: README.md (hunk 5 above)
Comment (ramius345) [reply README.md]: Updated the documentation.

**Comment by ramius345** [comment]
Rebased to lastest cop master.  Sorry I missed that.

**Comment by ramius345** [comment]
I'm going to convert this to helm before we merge it.  I'm not sure if we should reject this PR or just modify it.

**Comment by ramius345** [comment]
Since converting to helm, I am doing this on another branch.  I will create a different PR with the helm version.

**Pull request closed without merging**